Add score to user

stephendawsondev opened the pull request:
Add functionality to increase score - this still needs to be updated

stephendawsondev committed:
feat: add url path, add form to challenge page

stephendawsondev committed:
add functionality to increase score

stephendawsondev committed:
add set interval to variable for loading

stephendawsondev committed:
add correct score for found it and handle unfound

jdhernandezS1 reviewed (commented):
sry i said the db.sqlite3  file will not generate problems but yes.. 
please delete that file from your commit, not your files before merge because now its generating problems

stephendawsondev committed:
feat: skip button
- add skip button behaviour
- edit dialog modal on challenge.html
- style buttons on challenge.html

stephendawsondev committed:
Merge branch 'main' of https://github.com/stephendawsondev/Summer_Hackaton into add-score-to-user

stephendawsondev committed:
remove db.sqlite3 from versioning

stephendawsondev committed:
revert changes on this file - added by mistake
